src/DataBus/AzureDataBusPersistence.cs

```diff
@@ -28,6 +29,19 @@ protected override void Setup(FeatureConfigurationContext context)
             context.Services.AddSingleton(blobContainerClientProvider ?? new ThrowIfNoBlobServiceClientProvider());
             context.Services.AddSingleton<IDataBus>(serviceProvider => new BlobStorageDataBus(serviceProvider.GetRequiredService<IProvideBlobServiceClient>(),
                 dataBusSettings, new AsyncTimer()));
+            
+            context.Settings.AddStartupDiagnosticsSection(
+                "NServiceBus.Databus.AzureBlobStorage",
+                new
+                {
+                    dataBusSettings.UserProvidedConnectionString,
+                    UserProvidedBlobServiceClient = blobContainerClientProviderSetThroughSettings,
```

Comment:
Are we missing the container name?

Reply:
Also, if cleanup is disabled, what's the value of reporting things that do not matter?

Reply:
Good point